magento/pwa-studio
# Style icons with CSS

## Description
Style the `Icon` component with CSS rather than SVG attributes.

## Related Issue
PWA-445

## Acceptance 
- @tjwiebell 

### Verification Stakeholders
- @larsroettig 
- @sirugh 

### Specification

### Verification Steps
1. View icons throughout the site, verifying none have changed.
2. Fill and empty your cart, verifying that the icon changes when you do.

## Screenshots / Screen Captures (if appropriate)

## Checklist
* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
